### PR TITLE
Add env field to gelf

### PR DESCRIPTION
**Pull request by sbly**
https://themuse.atlassian.net/browse/SVC-412

### CODE REVIEW

**Comment by ian-axelrod** [comment]
Can you bump the version, too?

**Comment by sbly** [comment]
Okey dokey

**Comment by sbly** [comment]
@ian-axelrod Just FYI I can't merge this, in case you didn't realize

**Comment by sbly** [comment]
@ian-axelrod ping

File: muselog/__init__.py [hunk 1]
```diff
@@ -40,6 +40,10 @@ def setup_logging(root_log_level: Optional[str] = None,
             include_extra_fields=True,
             _app_name=os.environ["GRAYLOG_APP_NAME"]
         )
+        if os.environ.get("THEMUSE_ENV_LEVEL") == 2:
+            common_opts["_env"] = "production"
+        elif os.environ.get("THEMUSE_ENV_LEVEL") == 1:
```
Comment (ian-axelrod) [inline muselog/__init__.py]: Can you change this to use the new variables we talked about? GRAYLOG_ENV

Comment (sbly) [reply muselog/__init__.py]: done

**Comment by ian-axelrod** [comment]
Merging. You'll need to create and push a tag for this version.

**Pull request merged**